test/unit/webgl/p5.RendererGL.js

```diff
@@ -34,7 +34,8 @@ suite('p5.RendererGL', function() {
       assert.equal(myp5.webglVersion, myp5.WEBGL);
     });
 
-    test('works on p5.Graphics', function() {
+    // NOTE: should graphics always create WebGL2 canvas?
```

Comment:
I think the intended behaviour originally was like, graphics have their own `setAttributes()`, so you would make them be webgl 1 manually if you want it to be webgl 1, otherwise it'll try to be webgl 2 by default if it's available. I'd also be ok defaulting it to whatever the main canvas does too, since this is already very much an edge case.

Reply:
Ok, I see. I was a bit confused because it seemed like the main canvas and the graphics have different behavior dealing with the constants but I think I just missed the setAttributes. I think it's good to have it be different from the main canvas for more flexibility.

The test was failing for this case though so may need to check it later.